project-base/composer.json

```diff
@@ -112,8 +112,8 @@
         "phpstan/phpstan-symfony": "^1.1.8",
         "phpunit/phpunit": "^9.5.20",
         "psr/event-dispatcher": "^1.0.0",
-        "shopsys/coding-standards": "9.2.x-dev",
-        "shopsys/http-smoke-testing": "9.2.x-dev",
+        "shopsys/coding-standards": "10.0.0",
+        "shopsys/http-smoke-testing": "10.0.0",
```

Comment:
can you please also update `branch-alias` in all composer files, so the `master` branch will be installed with the `11.0.x-dev` version string?

Reply:
hm, I wonder why this is not automated, there is `UpdateBranchAliasReleaseWorker` in the `monorepo-builder` package, maybe we could leverage it, I will write this down as well.

Fixed in f94272008